src/Utils/resizer.ts

```diff
@@ -48,9 +48,10 @@ export const resize = (
     width?: number
     height?: number
     quality?: number
+    convert_to?: string
```

Comment:
Does Gemini have a specific set of strings that it allows for `convert_to`? If so, it'd be useful to type this property so that it only allows acceptable formats.

Reply:
I just did a quick look and it seem gemini doesn't have specified set of formats.